src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from "react";
+
+export default function Dropdown(props) {
+	const [isExpanded, setIsExpanded] = useState(true);
```

Comment:
Should it be expanded at the start? I think dropdowns should be collapsed initially.